README.md

```diff
@@ -94,6 +94,7 @@ docker run -d \
 | GIT_SYNC_ONE_TIME               | `--one-time`               | exit after the first sync                                                                                                                                                                                                                     | false                         |
 | GIT_SYNC_MAX_SYNC_FAILURES      | `--max-sync-failures`      | the number of consecutive failures allowed before aborting (the first sync must succeed, -1 will retry forever after the initial sync)                                                                                                        | 0                             |
 | GIT_SYNC_PERMISSIONS            | `--change-permissions`     | the file permissions to apply to the checked-out files (0 will not change permissions at all)                                                                                                                                                 | 0                             |
+| GIT_SPARSE_CHECKOUT            | `--sparse-checkout-file`         | the location of an optional [sparse-checkout](https://git-scm.com/docs/git-sparse-checkout#_sparse_checkout) file, same syntax as a .gitignore file.                                                                    | ""                             |
```

Comment:
Sorry - code says "GIT_SYNC_SPARSE_CHECKOUT_FILE"  :(

Reply:
You are 100% correct, I am clearly not awake enough to be making commits!